### DISCUSSION TITLE
Change TURN port number?

### DISCUSSION THREAD

**Discussion by brainrecall** [doitandbedone/ispyagentdvr-docker#212] [Q&A]
Hello, I'm looking to use this docker container on a system running several containers, one of them is for UniFi Controller. The UniFi Controller uses STUN on port 3478 to talk to all the network devices. I cannot change this port on this container, so it is already in use.

Is it possible to change the TURN port on the ispyagentdvr container and still have the agent work? When I tried changing the port to 3479, the Agent complained it was being blocked by a firewall.

**Comment by FA9US** [comment]
sure just run it behind Traefik and map it to some other port. alternatively XML config file to change the port or the run on docker compose

**Reply by brainrecall** [reply]
Ok I think I figured it out. I changed the port it was mapped to as 3479 both inside and outside the container. Then I started the container. I then stopped the container, and manually edited the newly created config/config.xml file. I changed the TURN port in that file to 3479. Then I could start the container and connect to the webpage. If I tried changing just the external container port, it would fail. I had to change both sides and the config.
